src/constants.ts

```diff
@@ -70,3 +70,7 @@ export const THUMBNAIL_SIZES = {
 export const DEFAULT_THUMBNAIL_SIZE = THUMBNAIL_SIZES.SMALL;
 
 export const FLAG_LIST_MAX_HEIGHT = 250;
+
+export const FAVORITE_ITEM_BUTTON_CLASS = 'favoriteButton';
+
+export const LIKE_ITEM_BUTTON_CLASS = 'likeButton';
```

Comment:
What's that used for? If it's for tests you should allow classes to be passed in the component, and pass as props.